website/docs/scaffolds/taco-shop.mdx

```diff
@@ -24,7 +24,7 @@ In a rush? You can follow the steps below to get up and running immediatley:
 
 ## Overview
 
-This scaffold implements a simple full stack Tezos project. It has a React dApp that interacts with a smart contract which stores the number of available_tacos and provides a function to buy tacos
+This scaffold implements a simple full stack Tezos project. It has a React dApp that interacts with a smart contract which stores the number of `available_tacos` and provides a function to buy tacos
```

Comment:
This scaffold implements a simple full stack Tezos project. It has a React dApp that interacts with a smart contract using Taquito. The contract stores the number of `available_tacos` and provides a function to buy tacos.